src/Pos/Update/Download.hs

```diff
@@ -33,34 +35,50 @@ import           Pos.WorkMode            (WorkMode)
 showHash :: Hash a -> FilePath
 showHash = toString . B16.encode . BA.convert
 
+-- TODO: if we're downloading update not for `cardano-sl`,
+-- but e. g. for `daedalus`, how do we check that version is new?
```

Comment:
Huh, that's a pretty good question indeed. I suppose that current check is correct and application name is "backend node" name, so there's no software app name "daedalus". Though i may be wrong. cc @georgeee